source/docs/contracts_deployment.md

```diff
@@ -45,13 +45,17 @@ module.exports = {
       "deployment": {
         <mark class="highlight-inline">"accounts": [
           {
-            "privateKey": "your_private_key"
+            "privateKey": "your_private_key" // Not recommended, as it may be checked into source control
           },
           {
-            "privateKeyFile": "path/to/file" // You can put more than one key, separated by , or ;
+            "privateKey": process.env.DAPP_PRIVATE_KEY // Better
           },
           {
-            "mnemonic": "12 word mnemonic",
+            // Ensure file isn't checked into source control
+            "privateKeyFile": "path/to/file" // Can contain more than one key, separated by , or ;
+          },
+          {
+            "mnemonic": "12 word mnemonic", // Not recommended, as it may be checked into source control
```

Comment:
I meant, you can use an environment variable for the mnemonic too

Reply:
I think this should mentioned as a note in the docs instead to ensure it's not in source control when using prod-accounts

Reply:
I’d argue a more in-your-face example (one you actually have to delete) gets the message through better. 

I didn’t replace the mnemonic example either because we already show how environmental variables can be pulled in just above and it’s better to have a good example of what a mnemonic looks like.

Reply:
It just looks weird and not clean to give the option and immediately say it's not 'recommended', using  a mnemonic in the config for dev purposes is fairly normal already (even for buffle). IMO it should be instead as a (hexo) note box with an example of using a require from some other file or env file.